src/components/pages/SingleReview.jsx

```diff
@@ -21,20 +23,39 @@ const SingleReview = () => {
       });
   }, [review_id]);
 
+  useEffect(() => {
+    if (review) {
+      getCommentsByReviewId(review_id)
+        .then((res) => {
+          setComments(res.data.comments);
+        })
+        .catch((err) => {
+          console.log(err);
+        });
+    }
+  }, [review, review_id]);
+
   if (isLoading) {
     return <p>Loading...</p>;
   }
 
   return (
-    <div className="game-review">
-      <img src={review.review_img_url} alt={review.title} />
-      <h2>{review.title}</h2>
-      <p>By: {review.owner}</p>
-      <p>Created at: {review.created_at}</p>
-      <p>{review.review_body}</p>
-      <p>Comments: {review.comment_count}</p>
-      <p>Votes: {review.votes}</p>
-      <p>Designer: {review.designer}</p>
+    <div className="single-review-page">
+      <div className="game-review">
+        <img src={review.review_img_url} alt={review.title} />
+        <h2>{review.title}</h2>
+        <p>By: {review.owner}</p>
+        <p>Created at: {review.created_at}</p>
+        <p>{review.review_body}</p>
+        <p>Comments: {review.comment_count}</p>
+        <p>Votes: {review.votes}</p>
+        <p>Designer: {review.designer}</p>
+      </div>
+      <div className="comments-section">
+        {comments.map((comment) => (
```

Comment:
In case there are no comments for a review, please indicate it to the user with a message like: 'No comments yet'.